packages/forma-36-react-components/src/components/Card/CardLoading/CardLoading.js

```diff
@@ -42,24 +63,31 @@ class CardLoading extends React.Component {
       children,
       testId,
       isLoading,
+      height,
       ...otherProps
     } = this.props;
 
     const classNames = cn(styles.CardLoading, extraClassNames, {
-      [styles['CardLoading--is-loading']]: isLoading,
+      [styles['CardLoading--is-loading']]: isLoading && !height,
     });
 
+    const Container = height ? React.Fragment : AnimateHeightContainer;
```

Comment:
this condition appears in multiple places (see L71, 74, 82). I think we should start to think about better abstraction.

e.g.
use inversion of control and passing container as prop

please, tell me what do you think